extensions/reviewed/PhysicsCar.json

```diff
@@ -2260,7 +2260,7 @@
           "type": "Number",
           "unit": "PixelAcceleration",
           "label": "Acceleration",
-          "description": "",
+          "description": "Object speed",
```

Comment:
but acceleration is not speed, that description is misleading in my opinion.
it should be more "How fast the object increases its speed"?